frontend/apps/webapp/src/features/menu/menuAdminService.ts

```diff
@@ -10,7 +10,7 @@ export interface MenuDto {
   url: string;
   position: number;
   children: MenuDto[];
-  authorities: string[];
+  authorities: any[];
```

Comment:
Why are we changing from string to any?

Reply:
Hi jeremy

Reason for changing string to any is, when I submit form I have to pass authorities as a array datatype and as response I get the authority as a string datatype, that's why I changed.

I will try to fix this again with same datatype.

Thanks

Reply:
Ok - maybe we should change the response type to match, so it will always be a string array.
I'll check the backend.